cmd/torusblk/flex.go

```diff
@@ -285,7 +285,10 @@ func mountAction(cmd *cobra.Command, args []string) {
 	// 		Device: mountdev,
 	// 	})
 	// }
-	os.MkdirAll(mountdir, os.ModeDir|0555)
+	if err := os.MkdirAll(mountdir, os.ModeDir|0555); err != nil && !os.IsExist(err) {
```

Comment:
err cannot be os.exist. see https://golang.org/pkg/os/#MkdirAll.

remove the !os.IsExist part, then lgtm. Thanks.